site/it/tutorials/quickstart/beginner.ipynb

```diff
@@ -0,0 +1,240 @@
+        "Questa breve introduzione usa [Keras](https://www.tensorflow.org/guide/keras/overview) per:\n",
+        "\n",
+        "1. Costruire una rete neurale che classifica immagini.\n",
+        "2. Allenare questa rete neurale.\n",
+        "3. E, infine, valutare la precisione del modello."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "hiH7AC-NTniF"
+      },
+      "source": [
+        "Questo è un [Google Colaboratory](https://colab.research.google.com/notebooks/welcome.ipynb) notebook file. Programmi python sono eseguiti run direttamente nel browser—un ottimo modo per imparare e utilizzare TensorFlow. Per seguire questo tutorial, esegui il file notebook in Google Colab cliccando sul bottone in cima a questa pagina.\n",
+        "\n",
+        "1. All'interno di Colab, connettiti al runtime di Python: In alto a destra della barra dei menu, seleziona *CONNECT*.\n",
+        "2. Esegui tutte le celle di codice di notebook: Seleziona *Runtime* \u003e *Run all*."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "nnrWf3PCEzXL"
+      },
+      "source": [
+        "Scarica e installa il package TensorFlow 2. Importa TensorFlow nel tuo codice:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "0trJmd6DjqBZ"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "# Install TensorFlow\n",
+        "try:\n",
+        "  # %tensorflow_version only exists in Colab.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
+        "\n",
+        "import tensorflow as tf"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "7NAbSZiaoJ4z"
+      },
+      "source": [
+        "Carica e prepara il [dataset MNIST](http://yann.lecun.com/exdb/mnist/). Converti gli esempi da numeri di tipo integer a floating-point:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "7FP5258xjs-v"
+      },
+      "outputs": [],
+      "source": [
+        "mnist = tf.keras.datasets.mnist\n",
+        "\n",
+        "(x_train, y_train), (x_test, y_test) = mnist.load_data()\n",
+        "x_train, x_test = x_train / 255.0, x_test / 255.0"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BPZ68wASog_I"
+      },
+      "source": [
+        "Costruisci il modello `tf.keras.Sequential` tramite livelli. Scegli un metodo di ottimizzazione e una funzione di perdita per l'allenamento:"
```

Comment:
Generally we don't traslate layer or loss in the day by day use in Italy (in the Deep learning domain)

Reply:
Thanks for the review.

You might consider adding a "Do not translate" list to the README, similiar to what some other languages are doing (for example, here's the [Korean do not translate list](https://github.com/tensorflow/docs/tree/master/site/ko#korean-translation-guide)). In ML in particular, it seems common to not translate some of these technical terms. But that's a community decision.

Reply:
Yes a do not translate list Is useful.

Reply:
For "loss function" I would ask your feedback on whether "funzione obiettivo" can be acceptable or I leave it as is in English. This also impacts the translation in `advanced.ipynb` where it is present as well.
Meanwhile I added a "Do not translate" section in the readme in Italian.